airbyte-integrations/connectors/source-klaviyo/source_klaviyo/manifest.yaml

```diff
@@ -154,12 +113,26 @@ definitions:
   profiles_stream:
     # Docs: https://developers.klaviyo.com/en/v2023-02-22/reference/get_profiles
     name: "profiles"
-    $ref: "#/definitions/base_incremental_stream"
+    $ref: "#/definitions/base_stream"
+    incremental_sync:
+      type: DatetimeBasedCursor
+      cursor_field: "updated"
+      start_datetime: "{{ config.get('start_date', '2012-01-01T00:00:00Z') }}"
+      datetime_format: "%Y-%m-%dT%H:%M:%S%z"
+      cursor_datetime_formats:
+        - "%Y-%m-%dT%H:%M:%S.%f%z"
+        - "%Y-%m-%dT%H:%M:%S%z"
+        - "%Y-%m-%d %H:%M:%S%z"
     schema_loader:
       type: InlineSchemaLoader
       schema: "#/definitions/profiles_schema"
     retriever:
       $ref: "#/definitions/profiles_retriever"
+      requester:
+        $ref: "#/definitions/profiles_retriever/requester"
+        request_parameters:
+          $ref: "#/definitions/profiles_retriever/requester/request_parameters"
+          "updated": "{{ stream_interval.start_time }}"
```

Comment:
I haven’t put too much thought into this but it seems like we should have something like:
```
requester:
  request_parameters:
    $ref: <...>
    filter: "greater-than({{ <cursor_field> },{{stream_interval.start_time}})"
    sort: {{ <cursor_field> }}
```

For `<cursor_field>` above, I'm thinking we could have the cursor field as a parameter and interpolate on this. This would avoid duplication (having the cursor_field in the `incremental_sync` component and in the `requester` component. That being said, there might be a better way to avoid this duplication but I'm unsure...